server/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@angular/language-service": "9.0.0-rc.9",
+    "vscode-languageclient": "6.0.0",
```

Comment:
why do we need language client in the server?

Reply:
We don’t. I just chose to update both at once.

Reply:
huh? but this is not updating the dep, it's *adding* a new dep. I'm wondering why this dep is needed.

Reply:
Did you mean to update `vscode-languageserver`, which is the dep below?

Reply:
Apologies. Should be fixed now.